src/voice/streamer.rs

```diff
@@ -318,6 +318,8 @@ pub fn ytdl(uri: &str) -> Result<Box<dyn AudioSource>> {
     Ok(pcm(true, ChildContainer(ffmpeg)))
 }
 
+/// Creates a streamed audio source from youtube search results with `youtube-dl`,`ffmpeg` and `ytsearch`
+/// Searches for video in youtube with a limit of 1
```

Comment:
Missing fullstop at the end of the sentence.
Also consider changing the sentence:
"Searches for the first listed YouTube video."

Reply:
Perhaps I'm splitting hairs here, but I suggest, "Takes the first video listed from a Youtube search."

Reply:
Yes, that's even better.
Maybe replace "a YouTube search" with "the YouTube search" since a search is already mentioned in the first doc line.